<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyProfile</class>
 <widget class="QDialog" name="ModifyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>369</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>369</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Modify Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>309</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_State">
      <property name="minimumSize">
       <size>
        <width>271</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>283</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Preffered Phone:     (</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_PhoneArea">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>)  -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_PhoneMiddle">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string> -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_PhoneRight">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>313</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Last">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>312</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>City:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_City">
      <property name="minimumSize">
       <size>
        <width>281</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>305</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Street Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_StreetAddress">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Update">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>314</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_First">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>315</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Zip Code:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ZipCode">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #686868;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Delete">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
